Test/git-issues/git-issue-1637.dfy

```diff
@@ -7,10 +7,10 @@ datatype Thing = Thing
 type Foo =  s: seq<uint8> | |s| < 10 witness *
 type Fii =  real 
 
-function Bar(t: Thing): Foo
-function Bii(t: Thing): Fii
+function {:axiom} Bar(t: Thing): Foo
+function {:axiom} Bii(t: Thing): Fii
 
-predicate Baz(f: Thing -> seq<uint8>)
+predicate {:axiom} Baz(f: Thing -> seq<uint8>)
```

Comment:
To my other point, this is especially confusing since `{:axiom}` DOESN'T mean this predicate is always true, it means all post-conditions always hold (which is vacuously true in this case). I think this emphasizes why not requiring it here as I suggest above is worth considering before we force all our users to add this.